fmriprep/workflows/bold/resampling.py

```diff
@@ -334,6 +353,39 @@ def _aslist(in_value):
     else:
         mask_mni_tfm.inputs.reference_image = template_out_grid
         bold_to_mni_transform.inputs.reference_image = template_out_grid
+
+    if freesurfer:
+    # Sample the parcellation files to functional space
```

Comment:
Comments are expected to be in line with the line below.

```suggestion
        # Sample the parcellation files to functional space
```

The reason is that the comment is considered "inside" the `if` block.